src/components/KRadio/KRadio.vue

```diff
@@ -73,7 +98,7 @@ const props = defineProps({
    */
   modelValue: {
     type: [String, Number, Boolean, Object],
-    default: 'on',
+    default: true,
```

Comment:
question: why would we default a radio input to being checked?

Reply:
Changed to `false`

Reply:
Just making sure -- will this break anything?

Reply:
It shouldn't because default value is always overwritten by the passed value. Technically as this is a required prop - default value can even be removed - do you think that would be better?

Reply:
Maybe yes?

Reply:
Done